docs/source/markdown/podman-create.1.md

```diff
@@ -739,6 +739,8 @@ This option conflicts with **--add-host**.
 
 Whether to disable OOM Killer for the container or not.
 
+This flag is not supported on cgroups V2 systems.
```

Comment:
this should be 'cgroup`, but after looking at the doc, we have both `cgroup` and `cgroups` all over the place.